src/main/java/com/amazon/ion/impl/IonReaderContinuableCoreBinary.java

```diff
@@ -2440,7 +2472,11 @@ public void consumeAnnotationTokens(Consumer<SymbolToken> consumer) {
             Marker marker = annotationTokenMarkers.get(i);
             if (marker.startIndex < 0) {
                 // This means the endIndex represents the token's symbol ID.
-                consumer.accept(getSymbolToken((int) marker.endIndex));
+                if (minorVersion == 1 && marker.typeId == IonTypeID.SYSTEM_SYMBOL_VALUE) {
+                    consumer.accept(new SymbolTokenImpl(SystemSymbols_1_1.get((int) marker.endIndex), -1));
```

Comment:
Another place where it will be better to store SymbolTokenImpls for each system symbol.

Reply:
Also note: SymbolTable.UNKNOWN_SYMBOL_ID is used in some places, but not here.